ci-operator/step-registry/gcp/provision/bastionhost/gcp-provision-bastionhost-commands.sh

```diff
@@ -89,6 +96,9 @@ CMD="gcloud compute instances create ${bastion_name} \
 if [ -n "${ATTACH_BASTION_SA}" ]; then
   CMD="${CMD} --service-account ${ATTACH_BASTION_SA} --scopes cloud-platform"
 fi
+if [[ "${OSD_QE_PROJECT_AS_SERVICE_PROJECT}" == "yes" ]]; then
```

Comment:
It just a bastion host, I guess not have to enable `--shielded-secure-boot`

Reply:
The OSD QE's project `openshift-qe-group-i-osd` has Secure Boot constraint enabled, so launch instance in that project has to enable `--shielded-secure-boot`. That's it, thanks!

Reply:
Got it.